### PR TITLE
Charmis/release flutter 3

### PR DESCRIPTION
**Pull request by CharmisG**
- Changed get started doc.
- Converted java to kotlin for example folder.

### CODE REVIEW

File: doc/get-started.md [hunk 1]
```diff
@@ -181,23 +181,26 @@ information on installing Reader SDK for iOS, see the
       with the photos stored on your mobile device.
 1. Update the `application:didFinishLaunchingWithOptions:` method in your app
```
Comment (hukid) [inline doc/get-started.md:182]: Please update the method name to the new swift method

Comment (hukid) [reply doc/get-started.md:182]: - [ ] update the method name to swift method name.

**Comment by hukid** [comment]
- [ ] the PR Description need to be updated to match the code change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ### v4.0.0 Sep 1st, 2022
 
 * Update package dependencies to latest versions for Null Safety support.
-* Update application UI 
+* Update example folder UI
```
Comment (hukid) [inline CHANGELOG.md:6]: remove "folder", we are updating the UI of the example, not the folder.

**Review by hukid — APPROVED** [review]

File: doc/get-started.md (hunk 1 above)
Comment (CharmisG) [reply doc/get-started.md:182]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (CharmisG) [reply CHANGELOG.md:6]: done

**Pull request merged**